index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <title> my portfolio</title>
+</head>
+
+<body>
+    <main class="container">
+        <header>
+            <div class="logo">logos</div>
+            <div class="humbuger"><img src="images/icones/Union.png" alt="humburger"></div>
```

Comment:
- [ ] Kindly make sure your logo image has an `<a>` tag in it, this is a project requirement. Also make sure your hamburger menu has a `<nav>` tag in it.